extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -86,26 +86,26 @@ static public function getFromFilter( $moduleId, $lang = null, $master = false,
 		wfProfileIn( __METHOD__ );
 
 		$timestampNow = wfTimestampNow();
-		$key = 'phalanx:' . $moduleId . ':' . ($lang ? $lang : 'all');
+		$key = 'phalanx:' . $moduleId . ':' . ( $lang ? $lang : 'all' );
 		$sLang = $lang ? $lang : 'all';
 		if ( $skipCache ) {
 			$blocksData = null;
-		} else if (isset(self::$moduleData[$moduleId][$sLang])) {
+		} else if ( isset( self::$moduleData[$moduleId][$sLang] ) ) {
 			$blocksData = self::$moduleData[$moduleId][$sLang];
 		} else {
-			$blocksData = $wgMemc->get($key);
+			$blocksData = $wgMemc->get( $key );
 		}
 
-		//cache miss (or we have expired blocks in cache), get from DB
-		if ( empty($blocksData) || (!is_null($blocksData['closestExpire']) && $blocksData['closestExpire'] < $timestampNow && $blocksData['closestExpire'])) {
+		// cache miss (or we have expired blocks in cache), get from DB
+		if ( empty( $blocksData ) || ( !is_null( $blocksData['closestExpire'] ) && $blocksData['closestExpire'] < $timestampNow && $blocksData['closestExpire'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "closestExpire" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)